fhir-model/src/main/java/com/ibm/fhir/model/util/ValidationSupport.java

```diff
@@ -604,19 +408,166 @@ public static void prohibited(Element element, String elementName) {
     }
 
     /**
-     * @throws IllegalStateExeption if the codeableConcept has coding elements that do not include codes from the required binding
+     * Check that the specified list of elements contain a code that is a member of the specified value set.
+     * 
+     * @param elements
+     *     the list of elements for which to check codes
+     * @param elementName
+     *     the name of the element
+     * @param valueSet
+     *     the URL of the value set to check membership against
+     * @param system
+     *     the value set system
+     * @param codes
+     *     the value set codes
+     * @throws IllegalStateExeption if each element in the list does not include a code from the required value set
      */
-    public static void checkCodeableConcept(CodeableConcept codeableConcept, String elementName, String valueSet, String system, String... codes) {
```

Comment:
This is a breaking change for this Java API. `fhir-model` is the one module where we are trying to keep some backwards-compatibility (e.g. see the project README at https://github.com/IBM/FHIR).
Can we deprecate that old methods and add new ones instead?

Reply:
Done.